guillotina/db/storages/pg.py

```diff
@@ -144,7 +144,7 @@ def _wrap_return_count(txt):
         NAIVE_UPSERT
         + """
 WHERE
-    tid = EXCLUDED.otid"""
+    {table_name}.tid = EXCLUDED.otid"""
```

Comment:
this is not used right now but while doing this PR I was testing the (non-naive) UPSERT and found that postgres was complaining about this column